modules/ROOT/pages/customize.adoc

```diff
@@ -0,0 +1,107 @@
+= Customize the Look and Feel of Your API Community
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+Use the https://help.salesforce.com/articleView?id=community_designer_overview.htm&type=5[Community Builder, role=external, window=blank] to customize the branding, look and feel, and site structure of your community pages to match your desired brand identity.
+
+Select *Community Builder* in the API Community Manager (ACM) administrator interface. By default, the
+Community Builder displays the *Home* page and the Pages drop-down list displays *Home*. You can select
+the Pages drop-down list to display a list of community pages and specify a different page for customization.
+
+The *Community Builder* interface contains the customization icons described here.
+
+== Components
+
+Select the *Components* lightning bolt icon in the top left navigation to display the component palette.
+The component palette includes Salesforce Community Cloud and ACM-specific components.
+Drag and drop the desired components from the component palette to the community page. Each
+component has its own configuration options that you can then access by selecting the component on the page.
+
+For detailed component information, refer to xref:acm-lightning-components.adoc[ACM Lightning Components] and
+the Salesforce Community Cloud https://help.salesforce.com/articleView?id=rss_component_reference_table.htm&type=5[Community Components, role=external, window=blank] documentation.
+
+Components that are typically used in *Home* pages include:
+
+* https://help.salesforce.com/articleView?id=rss_headline.htm&type=5[Headline, role=external, window=blank]
+* https://help.salesforce.com/articleView?id=rss_setup_nav_menu.htm&type=5[Navigation Menu, role=external, window=blank]: You can update *Max Items to Display (2-15)*,
+*Menu Item Alignment*, and use the *Navigation Menu* to display, add, remove, and customize options. *Home*
+is always present and is always the first item. The following properties can be customized:
+** *Name*
+** *Type*
+** *Page*
+** *Publicly available*
+* https://help.salesforce.com/articleView?id=rss_cms_collection.htm&type=5[CMS Collection, role=external, window=blank]
+* *API Carousel*: Displays APIs and API versions that are available to the community from Anypoint Exchange.
```

Comment:
ditto re. separating list item from its description. GLOBAL; I will stop marking in this set.

Reply:
Fixed throughout this file.